### DISCUSSION TITLE
troubleshooting webserver

### DISCUSSION THREAD

**Discussion by moziness** [firefly-iii/firefly-iii#5718] [Firefly III]
Context:  First time using Docker, installing Importer Tool (FIDI), the CLI output before I go back to my browser is below.

Symptom:  I try to go to Http://app:8080 and it cannot find the webpage.

Any ideas?  Thank you in advance.

CLI Output

Now in entrypoint.sh for Firefly III Data Importer
Script:        1.5 (2021-12-05)
User:          'root'
Group:         'root'
Working dir:   '/var/www/html'
Build number:  80
Build date:    22-01-2022 12:18:12 CET
Firefly III data importer v0.7.0
PHP: cli 8.0.13 Linux
Will now run Apache web server:
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.3. Set the 'ServerName' directive globally to suppress this message
AH00558: apache2: Could not reliably determine the server's fully qualified domain name, using 172.17.0.3. Set the 'ServerName' directive globally to suppress this message
[Sun Feb 06 01:57:30.331593 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.38 (Debian) configured -- resuming normal operations
[Sun Feb 06 01:57:30.331675 2022] [core:notice] [pid 1] AH00094: Command line: 'apache2 -D FOREGROUND'

**Comment by JC5** [comment]
Do you have a DNS entry for `app`?

**Comment by moziness** [comment]
I don't believe so. How do I create a DNS entry for app?  

I didn't need to create a DNS entry for the firefly app ( "localhost" ) and it seems to be working fine.

Thank you.

**Reply by JC5** [reply]
A wonderful exception, localhost.

This is your first Docker installation? 😉 

If you want to run multiple Docker containers, you must approach them on separate ports like so: `localhost:8080` or `localhost:80801`. 

You can also set up a reverse proxy that can handle this for you and then you can use virtual hosts like `http://app` or `http://fidi`.

**Reply by moziness** [reply]
I thought I tried localhost:8080, which didn't work, last night or this morning, but when I put in localhost:8081, it worked!

**Reply by moziness** [reply]
Unfortunately, when I go to import CSV now, the "Default import account" dropdown shows assets and liabilities, but both are unselectable by mouse or keyboard.  Have you heard of this issue?

**Reply by moziness** [reply]
Found it nevermind.  I needed to add some accounts in there first!

**Reply by moziness** [reply]
Thanks for making Firefly AND supporting it TODAY.  Love it so far.  Will support.

**Reply by JC5** [reply]
Glad to hear it works! let me know if you run into anything
